Changed docs.microsoft.com links to learn.microsoft.com Issue: #44175

atossell91 opened the pull request:
Fixes #44175

Used search and replace to correct the links. I have looked through them and all of them work.

Commit:
Updated links. Changed docs.microsoft to learn.microsoft

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=46206) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=46206) before we can accept your contribution.<br/><br/>:x: atossell91 [sign now](https://cla.dotnetfoundation.org/dotnet/aspnetcore?pullRequest=46206)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/aspnetcore?pullRequest=46206) it.</sub>

Commit:
Fixed some links that lead to 404 pages

dougbu commented on .editorconfig:
```diff
@@ -4,7 +4,7 @@
 ; Here are some resources for what's supported for .NET/C#
 ; https://kent-boogaart.com/blog/editorconfig-reference-for-c-developers
-; https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
+; https://learn.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017
```
nit
```suggestion
; https://learn.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference
```

atossell91 replied:
Yep, I checked through all of them last night.

dougbu replied:
great❕

dougbu commented on .github/fabricbot.json:
```diff
@@ -2652,7 +2652,7 @@
             "name": "addReply",
             "parameters": {
-              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [he ...
+              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [he ...
```
nit
```suggestion
              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [here](https://learn.microsoft.com/en-us/aspnet/core/blazor/debug?tabs=visual-studio#troubleshoot).\nIf these troubleshooting docs resolve your problem, please close the issue. Otherwise we'll follow up."
```

dougbu commented on README.md:
```diff
@@ -5,11 +5,11 @@ ASP.NET Core
 ## Get started
 
-Follow the [Getting Started](https://docs.microsoft.com/aspnet/core/getting-started) instructions in the [ASP.NET Core docs](https://docs.microsoft.com/aspnet/index).
+Follow the [Getting Started](https://learn.microsoft.com/aspnet/core/getting-started) instructions in the [ASP.NET Core docs](https://learn.microsoft.com/aspnet/index).
```
nit, I don't see the point of the final clause of this sentence. In addition, we should be consistent about `aspnet/core` (`aspnet/index` redirects to there).
```suggestion
Follow the [Getting Started](https://learn.microsoft.com/aspnet/core/getting-started) instructions.
```

atossell91 replied:
That makes sense! It should be good now.

dougbu commented on docs/AddingNewProjects.md:
```diff
@@ -51,4 +51,4 @@ while the second (the one with condition `'$(BuildMainlyReferenceProviders)' ==
 
 ## Manually saving solution and solution filter files
-VS is pretty good at keeping the files up to date and organized correctly. It will also prompt you if it finds an error and, in most cases, offer a solution to fix the issue. Sometimes just saving the file will trigger VS to resolve any issues automatically. However, if you would like to add a new solution filter file or update one manually you can find a tutorial link [here](https://docs.microso ...
+VS is pretty good at keeping the files up to date and organized correctly. It will also prompt you if it finds an error and, in most cases, offer a solution to fix the issue. Sometimes just saving the file will trigger VS to resolve any issues automatically. However, if you would like to add a new solution filter file or update one manually you can find a tutorial link [here](https://learn.micros ...
```
I'm not going repeat my comments about versions everywhere. The URIs all redirect to the latest view and that'll avoid having to update the docs whenever a new VS or .NET release happens.

atossell91 replied:
I took a look. I think I was able to version stuff, and I checked to make sure they all still work.

dougbu commented on eng/common/templates/job/job.yml:
```diff
@@ -2,7 +2,7 @@
 
 parameters:
-# Job schema parameters - https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#job
+# Job schema parameters - https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#job
```
Undo all changes under eng/common/; these files are overridden every time the Arcade SDK is updated.

dougbu replied:
This comment is not a nit. Please undo the changes in this part of the repo before we merge.

dougbu commented on src/Components/Samples/BlazorServerApp/Shared/MainLayout.razor:
```diff
@@ -6,7 +6,7 @@
 <div class="main">
     <div class="top-row px-4">
-        <a href="https://docs.microsoft.com/en-us/aspnet/" target="_blank">About</a>
+        <a href="https://learn.microsoft.com/en-us/aspnet/" target="_blank">About</a>
```
nit
```suggestion
        <a href="https://learn.microsoft.com/en-us/aspnet/core/" target="_blank">About</a>
```
This comment seems to apply in a fair number of changed files.

JamesNK committed:
Fix route syntax highlighting parameter color (#46208)

dougbu reviewed (commented):
I looked at about a quarter of the files. Have you confirmed the changed URIs are all valid❔ (Where I looked, I didn't find any broken links.)

Commit:
Reverted changes in eng/common

atossell91 committed:
Removed version numbers from some URLs

atossell91 committed:
Modified some links to include /core/

dougbu commented on src/Framework/AspNetCoreAnalyzers/src/Analyzers/RouteEmbeddedLanguage/RoutePatternClassifier.cs:
```diff
@@ -94,7 +94,12 @@ public void Visit(RoutePatternCatchAllParameterPartNode node)
+            // https://github.com/dotnet/roslyn/blob/e1ee2f544a7a4f8d536278bed4e180c54919276e/src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs#L181
+            //
+            // TODO: Fix properly with https://github.com/dotnet/aspnetcore/issues/46207
+            AddClassification(node.ParameterNameToken, "json - object");
```
Why are this file and src/Framework/AspNetCoreAnalyzers/test/RouteEmbeddedLanguage/RoutePatternClassifierTests.cs changing❔ Seems completely unrelated i.e. part of another (future) PR.

dotnet-maestro[bot] committed:
Update dependencies from https://github.com/dotnet/arcade build 20230117.5 (#46215)

[main] Update dependencies from dotnet/arcade

Suchiman committed:
Delete duplicate define (#46202)

The exact same thing exists 2 lines above that

mkArtakMSFT commented on .github/fabricbot.json:
```diff
@@ -2652,7 +2652,7 @@
             "name": "addReply",
             "parameters": {
-              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [he ...
+              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [he ...
```
Nice catch with the version-specific link. Thanks!

DamianEdwards replied:
typo

```suggestion
              "comment": "Thanks for contacting us.\nIt looks like this issue is related to debugging in Blazor WebAssembly application.\nBecause there are different components involved in providing debugging experience, we've put together a small troubleshooting guide that you can follow, to help us better understand where to direct this issue to, to get a faster resolution.\nYou can find it [here](https://learn.microsoft.com/en-us/aspnet/core/blazor/debug?tabs=visual-studio#troubleshoot).\nIf these troubleshooting docs resolve your problem, please close the issue. Otherwise we'll follow up."
```

mkArtakMSFT commented on eng/common/tools.ps1:
```diff
@@ -498,7 +498,7 @@ function InitializeXCopyMSBuild([string]$packageVersion, [bool]$install) {
 #   "components": ["componentId1", "componentId2", ...]
 #       Array of ids of workload components that must be available in the VS instance.
-#       See e.g. https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise?view=vs-2017
+#       See e.g. https://docs.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise
```
```suggestion
#       See e.g. https://learn.microsoft.com/en-us/visualstudio/install/workload-component-id-vs-enterprise
```

atossell91 replied:
I can change this, but I was told by dougbu to leave the links in this directory alone because they are changed automatically.

> Undo all changes under eng/common/; these files are overridden every time the Arcade SDK is updated.

mkArtakMSFT reviewed (approved):
Thanks for the PR. There is only one actionable comment that I left, where you've missed to update a link.

wtgodbe committed:
Don't depend on source-build job while it isn't running (#46220)

thaystg committed:
[blazor][wasm][debug]Press alt-shift-d and open firefox debug tab attached to the blazor app (#46132)

* Press alt-shift-d and open firefox debug tab attached to the blazor app

* remove debugger.launch.

* removing unrelated changes

* Removing unnecessary changes on chrome debugging.

* addressing @mkArtakMSFT comments

* Addressing @mkArtakMSFT comments.

* Addressing Steve comments, adding a console.warning message and remove the beautiful message, removed the Newtonsoft from the send message, todo: remove the Newtonsoft from receive message.

* Completely removing newtonsoft usage as asked by steve.

* Change warning message.

david-acker committed:
Move Kestrel shared source to src\Shared (#46112)

dependabot[bot] committed:
[main] (deps): Bump src/submodules/googletest (#46214)

Bumps [src/submodules/googletest](https://github.com/google/googletest) from `356fc30` to `ec25eea`.
- [Release notes](https://github.com/google/googletest/releases)
- [Commits](https://github.com/google/googletest/compare/356fc301251378e0f6fa6aa794d73714202887ac...ec25eea8f8237cf86c30703f59747e42f34b6f75)

---
updated-dependencies:
- dependency-name: src/submodules/googletest
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

surayya-MS committed:
Added withServerTimeout and withKeepAliveInterval to HubConnectionBuilder for java client (#46172)

* Added withServerTimeout and withKeepAliveInterval to HubConnectionBuilder for java client

Commit:
Implement RouteHandlerServices.Map and use in RequestDelegateGenerator (#46180)

* Implement RouteHandlerServices.Map and use in RequestDelegateGenerator

* Tweak generated code

* Remove warnings from generated code

* Fix docstring for RouteHandlerServices.Map

* Add baseline tests and fix global:: issue

* Clean up generated code

* Add HTTP verb caching and fix usings

* Use FQN and remove GeneratedCode attribute

marafiq committed:
Enumerate paramnames on startup (#46086)

Enumerate endpoint parameter names on startup instead of allocating a list.

atossell91 committed:
Fixed a typo abs->tabs

dougbu commented:
@atossell91 could you give us a reasonable commit message for when we squash and merge❔ I was about to enable auto-merge but GitHub is confusing your commits w/ those from recent 'main' PRs

atossell91 commented:
@dougbu Sure, how about "Updated links from microsoft.docs.* to microsoft.learn.* and removed version info from them". Do I need to do that on my end, or is that added when the merge happens?

dougbu commented:
> @dougbu Sure, how about "Updated links from microsoft.docs.* to microsoft.learn.* and removed version info from them". Do I need to do that on my end, or is that added when the merge happens?

I'll do it when I enable auto-merge. Resolving the conflicts first&hellip;

dougbu committed:
Merge branch 'main' into updating-links
